src/frontend/src/lib/derived/list-params.derived.ts

```diff
@@ -1,9 +1,15 @@
-import { listParamsStore } from '$lib/stores/list-params.store';
+import { LIST_PARAMS_CONTEXT_KEY, type ListParamsContext } from '$lib/types/list-params.context';
 import { nonNullish } from '@dfinity/utils';
+import { getContext } from 'svelte';
 import { derived, type Readable } from 'svelte/store';
 
-export const listParamsFiltered: Readable<boolean> = derived(
-	listParamsStore,
-	({ filter: { matcher, owner } }) =>
-		(nonNullish(matcher) && matcher !== '') || (nonNullish(owner) && owner !== '')
-);
+export const getDerivedListParamsFiltered = (): Readable<boolean> => {
+	const { store: listParamsStore }: ListParamsContext =
```

Comment:
This has only single use case in the code-base, not sure why this is outside the `CollectionEmpty` component. 

There is potential issue, if it is used somewhere else that has not created a list-params context, it will throw an error.

CC: @peterpeterparker

Reply:
Good point. I'll have a look and might replace it with a `$derived` variable scoped to the component.

Reply:
Done in #1682